runtime/rasdump/javadump.cpp

```diff
@@ -4619,18 +4676,29 @@ JavaCoreDumpWriter::writeThreadName(J9VMThread* vmThread)
 {
 	PORT_ACCESS_FROM_PORT(_PortLibrary);
 	if (NULL != vmThread) {
+#if JAVA_SPEC_VERSION >= 21
+		BOOLEAN isVThread = FALSE;
+		void *args[] = { _VirtualMachine, vmThread, &isVThread };
+#else /* JAVA_SPEC_VERSION >= 21 */
 		void *args[] = { _VirtualMachine, vmThread };
+#endif /* JAVA_SPEC_VERSION >= 21 */
```

Comment:
The first element, `_VirtualMachine` is unused by `protectedGetVMThreadName()`: please remove it.